kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -36,14 +34,14 @@ static ClusterCondition filterInvalid(String cluster, String filterName, String
                 String.format("Filter \"%s\" is invalid: %s", filterName, detail));
     }
 
-    public static ClusterCondition filterNotFound(String cluster, String filterName) {
+    public static ClusterCondition refNotFound(String cluster, LocalRef<?> ref) {
         return new ClusterCondition(cluster, ConditionType.Accepted, Status.FALSE, INVALID,
-                String.format("Filter \"%s\" does not exist.", filterName));
+                String.format("Resource of kind \"%s\" in group \"%s\" named \"%s\" does not exist.", ref.getKind(), ref.getGroup(), ref.getName()));
```

Comment:
Should we include namespace here as well?

Reply:
Maybe I was miss reading the diff but we weren't including kind or group either... 

I was actually musing if it would make mistakes easier to spot for users if we logged the full co-ordinates of what we were looking for. E.g. we can't find `kind: X in group: Y named: Z in namespace: A` makes it really obvious whats going on if you have created `kind: X in group: Y named: Z in namespace: B`. 

I guess what I'm saying its obvious to us but is it obvious to users? Is it actionable?

Reply:
I opened https://github.com/kroxylicious/kroxylicious/issues/1961 to discuss this.